docs/quickstart/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Quick start",
+    "position": 1
```

Comment:
we will want to update this to position 2 (and cascade all other positions too), since getting-started is in position 1. The only reason we don't get it rendered incorrectly right now is because if it's the same number, it loads in alphabetical order :)